File: print-manager-dotnet-core-vb/FaxExample.vb
Imports ceTe.DynamicPDF.Printing

Public Class FaxExample
    Public Shared Sub Run()
        Dim faxJob As New FaxPrintJob("My Fax Machine", "555-465-1177", Util.GetPath("Resources/DocumentA.pdf"))
        'faxJob.Print()
    End Sub
End Class

File: print-manager-dotnet-core-cs/FaxExample.cs
using ceTe.DynamicPDF.Printing;

namespace print_manager_dotnet_core_cs
{
    public class FaxExample
    {
        public static void Run()
        {
            FaxPrintJob faxJob = new FaxPrintJob("My Fax Machine", "555-465-1177", Util.GetPath("Resources/DocumentA.pdf"));
            faxPrintJob.Print();
        }
    }
}
